src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -0,0 +1,259 @@
+    Fig. 2 — Chrome DevTools showing the "displayed" state of a previously scheduled notification.
+  </figcaption>
+</figure>
+
+### Demo
+
+You can see Notification Triggers in action in the [demo][demo], which allows you to schedule
+notifications, list scheduled notifications, and cancel them. The source code is available on
+[Glitch][demo-source].
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="demo.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 3 — Notification Triggers <a href="https://notification-triggers.glitch.me/">demo</a>.
+  </figcaption>
+</figure>
+
+## Security and permissions
+
+We've designed and implemented the Notification Triggers API using the core principles defined in
+[Controlling Access to Powerful Web Platform Features][powerful-apis], including user control,
+transparency, and ergonomics. Because this API requires service workers,
+it also requires a secure context.
+Using the API requires the same permission as regular push notifications.
+
+### User control
+
+This API is only available in the context of a `ServiceWorkerRegistration`. This implies that all
+required data is stored in the same context and is automatically deleted when the service worker is
+deleted or the user deletes all site data for the origin. Blocking cookies also prevents service
+workers from being installed in Chrome, and therefore this API from being used.
+Notifications can always be disabled by the user for the site in site settings.
+
+### Transparency
+
+Unlike the Push API, this API does not depend on the network, which implies scheduled notifications
+need all required data beforehand, including image resources referenced by the `badge`,
+`icon` and `image` attributes. This means showing a scheduled notification is not observable by the
+developer and doesn't involve waking up the service worker until the user interacts with the
+notification. Consequently, there is currently no known way the developer could obtain information
+about the user through potentially privacy-invading approaches like IP address geolocation lookup.
+This design also allows the feature to optionally tap into scheduling mechanisms provided by
+the operating system like Android's
+[`AlarmManager`](https://developer.android.com/reference/android/app/AlarmManager), which helps
+preserve battery.
+
+## Feedback {: #feedback }
+
+We want to hear about your experiences with Notification Triggers.
+
+### Tell us about the API design
+
+Is there something about the API that doesn't work like you expected? Or are there missing methods
+or properties that you need to implement your idea? Have a question or comment on the security
+model? File a spec issue on the [Notification Triggers GitHub repo][issues], or add your thoughts to
+an existing issue.
+
+### Problem with the implementation?
+
+Did you find a bug with Chrome's implementation? Or is the implementation different from the spec?
```

Comment:
> Or is the implementation different from the spec?

@petele @tomayac @jpmedley there is no spec yet so this boilerplate text seems off. I suppose one will be coming out relatively soon, so we can leave this sentence in. But if we're going to use boilerplate code we should be careful that it actually applies in each scenario.

Reply:
> Did you find a bug with Chrome's implementation?

Also, why are we saying `Chrome` here instead of `Chromium`?

Reply:
Again a thing we should discuss in the article template. Commenting there, resolving here.